src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue

```diff
@@ -233,11 +253,12 @@ const updateDashboardData = async () => {
 const handleUpdateDashboardName = (name: string) => {
     dashboardDetailState.dashboardName = name;
 };
-const handleUpdateLabelList = (labelList: Array<string>) => {
-    dashboardDetailState.labelList = labelList;
+const handleUpdateLabelList = (labels: Array<string>) => {
+    dashboardDetailState.labels = [...labels];
 };
 const handleSave = async () => {
-    await updateDashboardData();
+    if (dashboardDetailState.dashboardId) await updateDashboardData();
+    if (dashboardDetailState.dashboardId === undefined) await createDashboard();
```

Comment:
nitpick: Are there any cases like `dashboardId === ''`? If not, then I think the `else` would enough.

Reply:
`dashboardId` must be always `undefined` when creating dashboard.